File: PHP/UberBlack.php
<?php
require_once('Car.php');
    class UberBlack extends Car {
        public $typeCarAccepted;
        public $seatMaterial;

        public function _construct($license, $driver, $typeCarAccepted, $seatMaterial) {
            parent::_construct($license,$driver);
            $this->typeCarAccepted = $typeCarAccepted;
            $this->seatsMaterial = $seatMaterial;
        }

    }


?>

File: Java/UberBlack.java
import java.util.Map;

class UberBlack extends Car {
    Map<String, Map<String, Integer>> typeCarAccepted;
    Map<String, Map<String, Integer>> seatsMaterial;

    public UberBlack(String license, Account driver, Map<String, Map<String, Integer>> typeCarAccepted, 
                    Map<String, Map<String, Integer>> seatsMaterial) {
            super(license, driver);
            this.typeCarAccepted = typeCarAccepted;
            this.seatsMaterial = seatsMaterial;
        }
    }
